vendor/github.com/go-jose/go-jose/v3/crypter.go

```diff
@@ -524,18 +526,18 @@ func (obj JSONWebEncryption) DecryptMulti(decryptionKey interface{}) (int, Heade
 		}
 	}
 
-	if plaintext == nil || err != nil {
+	if plaintext == nil {
```

Comment:
Note for other reviewers: this was part of a lint cleanup in 7d3b437c, presumably because the `err` being checked here isn't the `err` from the loop above (which is scoped to just the loop); it's an err that was declared and checked far above here. The cleanup is correct.